frontend/src/components/Status.tsx

```diff
@@ -42,18 +42,25 @@ export default function Status({
 }: {status: string, cluster?: any}) {
   const failedStatuses = new Set(['CREATE_FAILED', 'DELETE_FAILED', 'UPDATE_FAILED']);
 
-  const statusMap: Record<string, StatusIndicatorProps.Type> = {"CREATE_IN_PROGRESS": 'in-progress',
-    "CREATE_COMPLETE": 'success',
-    "CREATE_FAILED": 'error',
-    "DELETE_IN_PROGRESS": 'in-progress',
-    "DELETE_FAILED": 'error',
-    "RUNNING": 'success',
-    "STOPPED": 'error',
-    "STOP_REQUESTED": 'in-progress',
-    "UPDATE_FAILED": 'error',
-    "UPDATE_IN_PROGRESS": 'in-progress',
-    "UNKNOWN": 'error',
-    "UPDATE_COMPLETE": 'success'};
+  const statusMap: Record<string, StatusIndicatorProps.Type> = {'CREATE_IN_PROGRESS': 'in-progress',
+    'CREATE_COMPLETE': 'success',
```

Comment:
suggestion(non-blocking): it would be good to add a union type to the status so that who uses this component has the intellisense (and not just a `string` hint)